packages/iota/test/validation/aliasOutput.spec.ts

```diff
@@ -106,7 +106,7 @@ describe("Alias output validation", () => {
         expect(result.errors).toBeDefined();
         expect(result.errors?.length).toEqual(1);
         expect(result.errors).toEqual(expect.arrayContaining(
-            ["Alias output Unlock Conditions must be sorted in ascending order based on their Unlock Condition Type."]
+            ["Output Unlock Conditions must be sorted in ascending order based on their Unlock Condition Type."]
```

Comment:
Shouldn't we have "Alias output" at the start here to be inline with the rest of the errors ?

Reply:
The validateDistinct function got moved to validateUnlockConditions, and as a result, the error message is now general.